data/transactions/logic/assembler_test.go

```diff
@@ -1548,9 +1550,12 @@ func TestAssembleDisassembleCycle(t *testing.T) {
 			require.NoError(t, err)
 			none := testProg(t, notrack(t2), assemblerNoVersion)
 			require.Equal(t, ops.Program[1:], none.Program[1:])
-			t3 := "// " + t2 // This comments out the #pragma version
-			current := testProg(t, notrack(t3), AssemblerMaxVersion)
-			require.Equal(t, ops.Program[1:], current.Program[1:])
+			/*
```

Comment:
I imagine we'll want to do something to mark versions that intentionally changed bytecode, so we can keep the basic test, but disable it for some versions.

Reply:
I changed it so versions before multi-version use that test but with (mutli-version - 1) instead of the assembler max, but I don't see a great way of testing later versions until we get to a point where we won't be adding more multi-ops for several versions